internal/xdscache/v3/listener.go

```diff
@@ -419,13 +432,13 @@ func (c *ListenerCache) OnChange(root *dag.DAG) {
 			if vh.TCPProxy == nil {
 				var authFilter *http.HttpFilter
 
-				if vh.AuthorizationService != nil {
+				if vh.ExternalAuthorization != nil {
 					authFilter = envoy_v3.FilterExternalAuthz(
```

Comment:
now these fields have been consolidated into a struct, we could just pass that in to `FilterExternalAuthz` to reduce arguments